soh/src/overlays/actors/ovl_Bg_Breakwall/z_bg_breakwall.c

```diff
@@ -73,6 +94,8 @@ static InitChainEntry sInitChain[] = {
     ICHAIN_F32(uncullZoneDownward, 400, ICHAIN_STOP),
 };
 
+bool blueFireArrowsDC = false;
```

Comment:
having vars like this outside of functions (especially when they share a value with a single cvar) doesn't make the most sense to me

Reply:
My primary reason for using these bools in this way is as a "defense" against the player toggling the cvar on and off while these actors are loaded.  The modified collision gets set up when the actor is loaded and can't be reverted to its default collision without reloading the room.  I wanted to avoid the scenario where the actor had modified collision but no modified behavior, or vice versa.  

Essentially I'm not asking "Is the feature turned on?", I'm asking "Was the feature turned on when this actor was loaded?"

My goal was to always force the collision and the behavior to be in lockstep by setting the bool once at the same time the collider is set.  Off the top of my head it's probably not an issue for the red ice or mudwalls, but could be for sun switches since they're more complex and have on/off states and whatnot.

tl;dr I was trying to be cautious, perhaps overly so! :P

Reply:
gotcha! i'll resolve the other comments about this, would you mind adding some comments in code documenting that decision? maybe rename the var? (even something as verbose as `blueFireArrowsEnabledWhenActorLoaded` would work imo)

Reply:
Sure!  I'll make that a little clearer

Reply:
Renamed these vars and clarified some comments about them